public/toolkit/styles/molecules/_inputs.scss

```diff
@@ -383,3 +383,26 @@ input.icon-input-field {
   line-height: $icon-input-height;
   height: $icon-input-height;
 }
+
+// Search text field with x icon to clear
+// Javascript must be added to
+$clear-icon-margin: 13.5px;
+$clear-icon-height: 16px;
+.search {
+  display: inline-block;
+  position: relative;
+
+  input {
+    width: 15.5rem;
+  }
+  input.has-value {
+    // Center the icon within the padding. Subtract 1px to account for the border.
+    padding-right: rem-calc(($clear-icon-margin - 1) * 2 + $clear-icon-height);
```

Comment:
I'm not sure I understand why height is being used for a `right` calculation, is it because height and width are equal for the icon?

Since this is the only place it's used can we call the variable clear-icon-width or clear-icon-size?